e2etest/bgptests/bgp.go

```diff
@@ -1379,6 +1379,43 @@ var _ = ginkgo.Describe("BGP", func() {
 			}
 
 		})
+		ginkgo.It("FRR-MODE BGP Peer disable MP BGP", func() {
```

Comment:
I know that this test looks alike to the BGP Peer connect time test, i think in general its better to do a similar approach to the one in [frrk8s](https://github.com/metallb/frr-k8s/blob/89811e5b8238b2e0c83ee8d9382570efabc4c795/e2etests/tests/session.go#L65) where all session parameters are checked in the same test.
But because this test should not run for now in frrk8s mode i am leaving it in separate function to make the skipping easy.
i think we can combine them in the PR that adds the disable mp bgp in frrk8s mode (of course after we add the field to frrk8s) .
in general don't like code duplication but i think this option is temporary and after the tests will be combined will be a good option.

of course there are other options (creating function used by both tests or combining now and skip both of the tests in frrrk8s mode or different skipping method?) 
what do you think?

Reply:
It's totally fair to leave it as it is. We don't want to leave intermediate states where we are not testing the feature, and at the same time we can rework it later. Just file an issue so we won't forgot